djoffers/models.py

```diff
@@ -11,7 +11,7 @@ class HasOffersModel(models.Model):
     """
     Basic entity to store on our side. It points to remote HasOffers entity via integer ID.
     """
-    hasoffers_id = models.IntegerField(unique=True)
+    hasoffers_id = models.IntegerField(unique=True, null=True)
```

Comment:
Is it possible to have an object in HO without an ID? For me it doesn't make sense

Reply:
This diff shouldn't be here, removed. It is possible though. When you have a model that is being synced from local DB to HO.